<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OSG Bullet Example</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="OSGWidget" name="mOSGMainWidget" native="true">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>268</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>621</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="bInstructions">
           <property name="text">
            <string>Instructions</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bReset">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Number of Obstacles(10-50)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="numObstacles">
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>50</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Size of Obstacles (1-5)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sizeObstacles">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bGenerate">
           <property name="text">
            <string>Generate</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QGroupBox" name="verticalGroupBox">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
               <weight>75</weight>
               <bold>true</bold>
               <underline>false</underline>
              </font>
             </property>
             <property name="title">
              <string>Run</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QPushButton" name="bRun">
                <property name="font">
                 <font>
                  <pointsize>15</pointsize>
                  <weight>50</weight>
                  <bold>false</bold>
                  <underline>false</underline>
                 </font>
                </property>
                <property name="text">
                 <string>Manual</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="bAutoRun">
                <property name="font">
                 <font>
                  <pointsize>15</pointsize>
                  <weight>50</weight>
                  <bold>false</bold>
                  <underline>false</underline>
                 </font>
                </property>
                <property name="text">
                 <string>Auto</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>812</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionTest_Bullet">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSingle_Ball">
   <property name="text">
    <string>Single Ball</string>
   </property>
  </action>
  <action name="actionPyramid">
   <property name="text">
    <string>Pyramid</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionGrid">
   <property name="text">
    <string>Grid</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>OSGWidget</class>
   <extends>QWidget</extends>
   <header>osgwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
